src/firestore/Cargo.toml

```diff
@@ -40,7 +40,7 @@ tonic       = { version = "0.13", features = ["tls-native-roots", "tls-ring"] }
 tokio       = { version = "1", features = ["macros", "rt-multi-thread"] }
 tracing     = "0.1"
 # Local crates
-auth = { version = "0.18", path = "../auth", package = "google-cloud-auth" }
+auth = { version = "0.19", path = "../auth", package = "google-cloud-auth" }
```

Comment:
I am assuming this library is special because it is hand written?

Why does it depend on auth directly?

Reply:
It is a little hand written.... At the moment it uses auth directly, because I have not completed the implementation of a generic `gax-internal::grpc::Client`